Itm 1042

dereknop opened the pull request:
[Ticket](https://nextcentury.atlassian.net/jira/software/projects/ITM/boards/116?assignee=60ba425da547eb00686ee0ce&jql=assignee%20IN%20%2860ba425da547eb00686ee0ce%2C%20empty%29&selectedIssue=ITM-1042)

http://localhost:3000/participant-progress-table

A lot of funkiness with the progress table should now be cleaned up:

- Phase 2 table should now have a del-5 column
- Text based scenario start and end times are properly being pulled (before it was grabbing the start and end times from the same scenario)
- The .xlsx download should now separate the two phases and only include the headers for that phase. These column headers should be in the same order as in the UI and the highlighting behavior should carry over into the download
- You shouldn't see any phase 1 fields (e.g `MJ-1`) in the phase 2 data and vice versa

jaudick reviewed (approved):
The ui and downloaded csv tables all look as described

nextcen-dgemoets reviewed (commented):
It looks like you also changed the Download Filtered Data to be local to a specific Phase, which is good!  It's only present if you've filtered the current Phase of data, whereas before data was always filtered because you either had Phase 1 or Phase 2 selected.  That said, I wonder if the "Showing x of y rows based on filters" line should change accordingly-- that is, if you're looking at unfiltered Phase 1 (or 2) data, this line doesn't show.  Then if you apply a filter, the denominator is the count for the current phase.

Also, not part of this PR I guess, but the "All Text (5)" filter should be changed to 4 for Phase 2, no, since that's the max?

dereknop commented:
> It looks like you also changed the Download Filtered Data to be local to a specific Phase, which is good! It's only present if you've filtered the current Phase of data, whereas before data was always filtered because you either had Phase 1 or Phase 2 selected. That said, I wonder if the "Showing x of y rows based on filters" line should change accordingly-- that is, if you're looking at unfiltered Phase 1 (or 2) data, this line doesn't show. Then if you apply a filter, the denominator is the count for the current phase.
> 
> Also, not part of this PR I guess, but the "All Text (5)" filter should be changed to 4 for Phase 2, no, since that's the max?

Agree on both fronts, updated

nextcen-dgemoets reviewed (approved):
LGTM!
